tox.ini

```diff
@@ -26,7 +26,7 @@ deps =
     mpl32: matplotlib==3.2.*
     mpl33: matplotlib==3.3.*
     mpl34: matplotlib==3.4.*
-    mpl35: matplotlib==3.5.*
+    mpl35: matplotlib==3.5.1
```

Comment:
@ConorMacBride We should make an issue to unpin this, right?

Reply:
We no longer need to change this.